.github/workflows/ci.yml

```diff
@@ -12,89 +12,23 @@ permissions:
 defaults:
   run:
     shell: bash
-
+      
 concurrency:
   group: ${{ github.ref }}
 
 env:
   ACTIONS_ALLOW_UNSECURE_COMMANDS: true
 
 jobs:
-  set-runner-image-name:
-    name: Set Runner Image Name
-    runs-on: ubuntu-22.04
-    steps:
-      - name: Checkout Branch
-        uses: actions/checkout@v3
-
-      - name: Set Runner Image
-        id: set_runner_image
-        run: echo "runner_image_name=ghcr.io/craftsmenltd/bloodconnect-ci-image:$(md5sum ./Dockerfile | awk '{ print $1 }')" >> $GITHUB_OUTPUT
-    outputs:
-        runner_image_name: ${{ steps.set_runner_image.outputs.runner_image_name }}
-
   handle-docker-image:
-    name: Handle Docker Image
-    needs: set-runner-image-name
+    name: 🐳 Handle Docker Image
```

Comment:
what does it mean by handle docker image? are we setting up the environment or updating it ? it's not clear from the name.

Reply:
Updated in f7270e29dddbc6b96c720aaca7302044b7b08e54